src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -178,7 +204,16 @@ class EmojiPickerMenu extends Component {
 
         // Arrow Down enable arrow navigation when search is focused
         if (this.searchInput && this.searchInput.isFocused() && this.state.filteredEmojis.length) {
-            if (arrowKey !== 'ArrowDown') {
+            if (['ArrowDown', 'ArrowRight'].indexOf(arrowKey) === -1) {
```

Comment:
Can this be changed to something like `arrowKey !== 'ArrowDown' || arrowKey !== 'ArrowRight'`